docs/visualization/visualize_lfp_responses.ipynb

```diff
@@ -84,7 +84,7 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    print("There are", len(all_stim_times), "stimulus times from rows that satisfy the condition.")
</pre><p>good clarification. This will do well to explain the <em>why</em> this code is here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>